pkg/netflow/portrollup/portrollup.go

```diff
@@ -151,9 +153,20 @@ func (prs *EndpointPairPortRollupStore) UseNewStoreAsCurrentStore() {
 	defer prs.mu.Unlock()
 
 	prs.curStore = prs.newStore
-	prs.newStore = make(map[string]*portRollupTracker)
+	prs.newStore = make(map[string][]uint16)
 }
 
-func buildStoreKey(sourceAddr []byte, destAddr []byte) string {
-	return string(sourceAddr) + "|" + string(destAddr)
+func buildStoreKey(sourceAddr []byte, destAddr []byte, endpointT endpointType, port uint16) string {
```

Comment:
@AlexandreYang could help document this with a comment on how we build the key

Is I read it correctly:

* given a port is a 2 bytes value (uint16)
* you take MSB and LSB over the 2 bytes a port is made of (portPart1, portPart2)
* then you stringify sourceAddr, destAddress, endpointT and port's MSB and LSB

What about using uint16 to Hex instead to stringify ?

Reply:
Oh wow, did not consider the overhead for hex ! Thank you, learnt something